### PR TITLE
minor updates to frontend 

### PR DESCRIPTION
**Pull request by yerx**
Please ignore the branch name. This PR is just for minor changes. I will be heading to my proxy repo to work on the proxy server.

### CODE REVIEW

File: client/components/MediaDisplay.jsx [hunk 1]
```diff
@@ -12,31 +12,31 @@ background: black;
...
         </div>
         <div className="image-cover">
-          {/* <img src={this.props.images[1]} alt="an image of something" /> */}
-          <img src="https://picsum.photos/200/300" />
+          <img src={props.images[1]} alt="an image of something" />
+          {/* <img src="https://picsum.photos/id/1011/200/300" /> */}
         </div>
         <div className="vertical-stack-images">
           <div className="image-cover-stacked">
-            {/* <img src={this.props.images[2]} alt="an image of something" /> */}
```
Comment (kswanson25) [inline client/components/MediaDisplay.jsx]: Did you find a way to make photos fill their entire div containers?

Comment (kswanson25) [reply client/components/MediaDisplay.jsx]: just wondering if it ended up working out! ^^

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
-# Media-Carousel-Service
\ No newline at end of file
+# Media-Carousel-Service
+
+## Getting Started
+- Run `npm install`
+- Open two terminal windows. Start up the server with `npm run start:dev` and start up webpack with `npm run build:dev`
```
Comment (kswanson25) [inline README.md]: LGTM, very clean and easy to understand ReadMe.

File: client/app.jsx [hunk 3]
```diff
@@ -97,6 +74,26 @@ class App extends React.Component {
       });
   }
 
+  checkNavAndFooter() {
```
Comment (ga-possum) [inline client/app.jsx]: This is all great! Especially how you send these methods down to components!

File: client/components/MediaDisplay.jsx (hunk 1 above)
Comment (yerx) [reply client/components/MediaDisplay.jsx]: Yes, I mostly figured it out. Via CSS, for each image, I set it to `object-fit: cover; width: 100%; height: 100%`

File: client/app.jsx (hunk 3 above)
Comment (yerx) [reply client/app.jsx]: Thanks!

**Pull request merged**